src/main/java/it/pagopa/ecommerce/commons/domain/EmptyTransaction.java

```diff
@@ -42,9 +48,15 @@ private TransactionActivated applyActivation(TransactionActivatedEvent event) {
     private TransactionActivationRequested applyActivationRequested(TransactionActivationRequestedEvent event) {
         return new TransactionActivationRequested(
                 new TransactionId(UUID.fromString(event.getTransactionId())),
-                new RptId(event.getRptId()),
-                new TransactionDescription(event.getData().getDescription()),
-                new TransactionAmount(event.getData().getAmount()),
+                event.getData().getNoticeCodes().stream()
+                        .map(
+                                n -> new NoticeCode(
+                                        new PaymentToken(null),
+                                        new RptId(n.getRptId()),
+                                        new TransactionAmount(n.getAmount()),
+                                        new TransactionDescription(n.getDescription())
+                                )
+                        ).collect(Collectors.toList()),
```

Comment:
```suggestion
                event.getData().getNoticeCodes(),
```

Reply:
They are not the same objects. One is from domain package, the other are from documents package